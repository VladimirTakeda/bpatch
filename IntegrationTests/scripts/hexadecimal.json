{
  "dictionary": {
    "hexadecimal": {
      "should": ["73", "68", "6f", "75", "6c", "64"],
      "must": ["6d", "75", "73", "74"],
      "processing": ["70", "72", "6F", "63", "65", "73", "73", "69", "6E", "67", "2E"],
      "fileprocessing": ["66", "69", "6C", "65", "20", "70", "72", "6F", "63", "65", "73", "73", "69", "6E", "67", "2E", "20", "41", "6E", "64", "20", "69", "74", "20", "69", "73", "20", "68", "65", "72", "65", "2E"]
    }
  },
  "todo": [
    {
      "replace": {
        "should": "must",
        "processing": "fileprocessing"
      }
    }
  ]
}