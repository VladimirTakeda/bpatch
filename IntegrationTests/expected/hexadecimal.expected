Some text to replace.
Here must be another result
after file processing. And it is here.